qa/L0_batcher/batcher_test.py

```diff
@@ -59,7 +59,7 @@
 _deferred_exceptions = []
 
 
-class BatcherTest(unittest.TestCase):
+class BatcherTest(tu.TestResCollector):
```

Comment:
Don't use non-standard abbreviations in names when possible, so use TestResultCollector